tests/Microsoft.DotNet.Framework.Docker.Tests/EnvironmentVariableInfo.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.DotNet.Framework.Docker.Tests
+{
+    public class EnvironmentVariableInfo
```

Comment:
I think env var tests should be added for WCF and ASP.NET as well.  If not now, please log an issue to track the work.

Reply:
Logged issues:
* https://github.com/microsoft/dotnet-framework-docker/issues/455
* https://github.com/microsoft/dotnet-framework-docker/issues/456